From b796da9eb6eef9ee8ab2059125ebe51a116c8995 Mon Sep 17 00:00:00 2001
From: Erwan Le Ray <erwan.leray@foss.st.com>
Date: Fri, 27 Mar 2020 15:15:01 +0100
Subject: [PATCH 126/529] ARM: dts: stm32: activate usart3 for stm32mp157x-ev1
 a7-examples

Activate usart3 (available on GPIO expansion connector) for
stm32mp157x-ev1 a7-examples.

Signed-off-by: Erwan Le Ray <erwan.leray@foss.st.com>
Change-Id: Ibc2261357ca985e51b7c359737d6ae311da552f4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277722
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277723
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277724
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277725
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Tested-by: Valentin CARON <valentin.caron@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157a-ev1-a7-examples.dts | 4 ++++
 arch/arm/boot/dts/stm32mp157c-ev1-a7-examples.dts | 4 ++++
 arch/arm/boot/dts/stm32mp157d-ev1-a7-examples.dts | 4 ++++
 arch/arm/boot/dts/stm32mp157f-ev1-a7-examples.dts | 4 ++++
 4 files changed, 16 insertions(+)

--- a/arch/arm/boot/dts/stm32mp157a-ev1-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157a-ev1-a7-examples.dts
@@ -13,3 +13,7 @@
 	model = "STMicroelectronics STM32MP157A-EV1 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157a-ev1-a7-examples", "st,stm32mp157a-ev1", "st,stm32mp157a-ed1", "st,stm32mp157";
 };
+
+&usart3 {
+	status = "okay";
+};
--- a/arch/arm/boot/dts/stm32mp157c-ev1-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157c-ev1-a7-examples.dts
@@ -13,3 +13,7 @@
 	model = "STMicroelectronics STM32MP157C-EV1 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157c-ev1-a7-examples", "st,stm32mp157c-ev1", "st,stm32mp157c-ed1", "st,stm32mp157";
 };
+
+&usart3 {
+	status = "okay";
+};
--- a/arch/arm/boot/dts/stm32mp157d-ev1-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157d-ev1-a7-examples.dts
@@ -13,3 +13,7 @@
 	model = "STMicroelectronics STM32MP157C-EV1 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157d-ev1-a7-examples", "st,stm32mp157d-ev1", "st,stm32mp157d-ed1", "st,stm32mp157";
 };
+
+&usart3 {
+	status = "okay";
+};
--- a/arch/arm/boot/dts/stm32mp157f-ev1-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157f-ev1-a7-examples.dts
@@ -13,3 +13,7 @@
 	model = "STMicroelectronics STM32MP157F-EV1 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157f-ev1-a7-examples", "st,stm32mp157f-ev1", "st,stm32mp157f-ed1", "st,stm32mp157";
 };
+
+&usart3 {
+	status = "okay";
+};
